helpers/nullauthhandler.go

```diff
@@ -34,6 +35,12 @@ func (h *NullAuthHandler) Change(fs billy.Filesystem) billy.Change {
 	return nil
 }
 
+// List of all exported file systems.
+func (h *NullAuthHandler) Export(context.Context) []nfs.Export {
```

Comment:
the directory here won't be at '/mount', so this seems wrong? - the default behavior should either return no exports when asked, or an rpc error to indicate the method isn't implemented?

Reply:
The `/mount` was taken from the documentation (README), it could be named something else?

i.e. `localhost:/mount`

Reply:
For the normal NFS service, it returns:

```
$ nfs-ls -D nfs://localhost
nfs://localhost/srv/homes
```

And then fail to mount anything not listed.

`Failed to mount nfs share`